dspace/config/spring/rest/README.md

```diff
@@ -0,0 +1 @@
+You should only add Spring XML definition files here if there is really no way to load them through automatic component scanning.
```

Comment:
A good use case to place xml here is to allow override of the provided configuration such as the listners enabled for usage events that you have included

Reply:
I hope that in the future we also find an easy way (using XML?) to override the default bean implementations we now have in dspace-spring-rest. For example, what if I want to use a custom `ItemConverter` bean in my own custom repository.